commands/build_test.go

```diff
@@ -62,7 +62,7 @@ func Test_parseBuildArgs_EmptyKey(t *testing.T) {
 }
 
 func Test_parseBuildArgs_MultipleSeparators(t *testing.T) {
```

Comment:
@s8sg the test name should change from parseBuildArgs to `parseMap` in this instance right?

Reply:
@alexellis Right. I'll update